vars/uploadMinSnapshotsToS3.groovy

```diff
@@ -30,7 +30,7 @@ void call(Map args = [:]) {
     sh """
         cp ${srcDir}/${baseName}.${extension} ${srcDir}/${baseName}-latest.${extension}
         cp ${srcDir}/${baseName}.${extension}.sha512 ${srcDir}/${baseName}-latest.${extension}.sha512
-        sed -i "s/.${extension}/-latest.${extension}/g" ${srcDir}/${baseName}-latest.${extension}.sha512
+        sed -i '' "s/.${extension}/-latest.${extension}/g" ${srcDir}/${baseName}-latest.${extension}.sha512
```

Comment:
FYI: on OSX there's `gsed` which works like *nix sed, otherwise it's a world of pain if you run into more problems

Reply:
Got it @dblock thanks will create the MacOS agent `AMI` with `gsed` part of it.

Reply:
AMI related PR https://github.com/opensearch-project/opensearch-ci/pull/175/files